readthedocs/templates/errors/proxito/404/no_project_page.html

```diff
@@ -0,0 +1,10 @@
+{% extends "errors/proxito/404/base.html" %}
+{% load i18n %}
+
+{# project exists but page doesn't #}
+
+{% block 404_error_message %}
+  {% include "errors/proxito/404/include_error_message.html" %}
+  {% include "errors/proxito/404/include_search.html" with search_project=project.slug %}
+  {% include "errors/proxito/404/include_tips.html" %}
+{% endblock %}
```

Comment:
All these blocks should use `endblock <block name>`.

Reply:
I didn't alter these templates besides changing paths. If we want to alter these templates, that would be separate work I feel.

And are you thinking of the tests on ext-theme? Are we using djlint on the old templates now? It doesn't seem like a strong priority to be lint fixing our templates we're deprecating either way.

Reply:
Oh, no. I don't care about old templates. I thought they were for ext-theme.